content/terraform-docs-common/docs/cloud-docs/registry/publish-modules.mdx

```diff
@@ -5,123 +5,198 @@ description: Use the HCP Terraform private registry to publish and share private
 
 [vcs]: /terraform/cloud-docs/vcs
 
-# Publish private modules to the HCP Terraform private registry
+# Publish private modules to the private registry
+
+This topic describes how to publish private modules to an organization's private registry. For instructions on adding modules from the public registry, refer to [Add modules from the Terraform registry](/terraform/cloud-docs/registry/add).
 
 > **Hands-on:** Try the [Share Modules in the Private Module Registry](/terraform/tutorials/modules/module-private-registry-share) tutorial.
 
-In addition to [adding modules from the Terraform Registry](/terraform/cloud-docs/registry/add), you can publish private modules to an organization's HCP Terraform private registry. The registry handles downloads and controls access with HCP Terraform API tokens, so consumers do not need access to the module's source repository, even when running Terraform from the command line.
+## Introduction
+
+When publishing modules to the private registry, the registry downloads the module source code. It uses API tokens to control access so consumers do not need access to the module's source repository, even when running Terraform from the command line. Additionally, the private registry uses your [version control system (VCS) integrations][vcs] and delegates management tasks, such as handling new version releases, to your VCS provider. The only manual tasks are adding a new module and deleting module versions.
+
+### Access to private modules
+
+Private modules are only available to members of the organization where you add them. In Terraform Enterprise, they are also available to organizations that you configure to [share modules](/terraform/enterprise/admin/application/registry-sharing) with that organization.
```

Comment:
```suggestion
Private modules are only available to members of the organization you publish them to. In Terraform Enterprise, they are also available to organizations that you configure to [share modules](/terraform/enterprise/admin/application/registry-sharing) with that organization.
```

Since we describe this action as "publish" above.